app/code/Magento/Pwa/Test/Mftf/Test/VeniaCartPageGuestCheckout.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<!--Add 2 Products to Cart-->
+<!--Go To Cart Page-->
+<!--check products exists with same configuration-->
+<!--Apply Shiiping-->
+<!--Apply Coupon Code-->
+<!--Place holder for checkout-->
+
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="CartPageGuestCheckout">
+        <annotations>
+            <features value="PWA"/>
+            <stories value="Cart Page"/>
+            <title value="Venia guest user should be able to checkout via Cart Page"/>
+            <description value="Venia guest user should be able to checkout via Cart Page"/>
+            <severity value="CRITICAL"/>
+            <testCaseId value="MC-31971"/>
+            <group value="PWA"/>
+            <group value="Cart"/>
+        </annotations>
+
+        <before>
+            <actionGroup ref="GoToVeniaDesktopStorefront" stepKey="goToVeniaDesktopStorefront1"/>
+
+            <actionGroup ref="GoToProductDetailsPage" stepKey="goToProductDetailsPage">
+                <argument name="product" value="ValeriaTwoLayerTank"/>
+            </actionGroup>
+
+            <actionGroup ref="SetProductColorAndSizeAndQuantityOptions" stepKey="addProductToCartWithColorAndSizeAndQuantityOptions1"/>
+            <actionGroup ref="TemporaryWait3" stepKey="waitForElementToLoad"/>
+            <actionGroup ref="AddToCart" stepKey="addToCart1"/>
+
+            <actionGroup ref="GoToProductDetailsPage" stepKey="goToProductDetailsPage2">
+                <argument name="product" value="IsadoraSkirt"/>
+            </actionGroup>
+
+            <actionGroup ref="SetProductColorAndSizeAndQuantityOptions" stepKey="addProductToCartWithDifferentConfigurations">
+                <argument name="product" value="IsadoraSkirt"/>
```

Comment:
Is this to identify which product to the options on?

Reply:
Yes, it refers the product from `Data/VeniaProductsData.xml`